src/explorer/InvalidWebAppTreeItem.ts

```diff
@@ -0,0 +1,25 @@
+import * as path from 'path';
```

Comment:
Please add the Microsoft copyright header - I'm not an expert on the legalese but you can check out this section for more info: https://github.com/Microsoft/vscode-azureappservice#contributing